task/aws-cli/0.2/aws-cli.yaml

```diff
@@ -19,7 +19,7 @@ spec:
       optional: true
     - name: secrets
       optional: true
-      mountPath: /tekton/home/.aws
```

Comment:
hi @aokugel, changing this to `/root` will lead to issues for those who are running these tasks as non-root

Reply:
is there any workaround apart from changing the mountPath to `/root` ?

Reply:
I believe it may be possible to change the directory that the aws-cli image is looking for the config/credentials using an environment variable:  

https://github.com/aws/aws-cli/issues/5120#issuecomment-1958570569

I can look into this.

Reply:
by utilizing a [stepTemplate](https://tekton.dev/docs/pipelines/tasks/#specifying-a-step-template) we can set the `HOME` environment variable to `/tekton/home`, which allows the aws config/credentials to work as expected.  Would this be a satisfactory solution?

Reply:
yes @aokugel, this can be done